deployment/config.go

```diff
@@ -77,9 +87,17 @@ func (c *Config) IsValid() error {
 		return fmt.Errorf("load-test package file must be a tar.gz file")
 	}
 
+	if c.HasElasticSearch() && c.VpcID == "" {
```

Comment:
I also got this

```
Error: "domain_name" must start with a lowercase alphabet and be at least 3 and no more than 28 characters long. Valid characters are a-z (lowercase letters), 0-9, and - (hyphen)
```

when I gave "agnivaES" as the name. Might be worth improving the check on L80 to incorporate the length limit and lowercase restriction. 0/5 on whether you want to do it now or in a separate PR.

Reply:
Done in this PR